vendor/DOtherSide/lib/src/Status/KeychainManager.mm

```diff
@@ -41,29 +41,27 @@
 
 void KeychainManager::readDataAsyncMacOs(const QString& key)
 {
-    auto readAction = [this, key](){
+    // auto readAction = [this, key](){
```

Comment:
Can we just remove the whole keychain-related code from `DOtherSide`?

Reply:
Not yet, only when we update the usage everywhere to the new `Keychain.qml`.
Also, this specific change shouldn't be here, it was for debugging purposes, I'll remove it.